### PR TITLE
Updated example inputs in spec and help of urlscan

### PR DESCRIPTION
**Pull request by tylerslijboom-r7**
### Description

Describe the proposed changes:

  - Example inputs added to plugin spec and help.md

### Testing

```
icon-validate . -a
```

### CODE REVIEW

File: urlscan/help.md [hunk 1]
```diff
@@ -267,6 +295,7 @@ _This plugin does not contain any troubleshooting information._
 
 # Version History
 
+* 2.1.5 - Update to add example input
```
Comment (jschipp-r7) [inline urlscan/help.md:298]:
```suggestion
* 2.1.5 - Add example inputs
```

File: urlscan/help.md [hunk 2]
```diff
@@ -20,9 +20,9 @@ on potential malicous indicators.
 
 The connection configuration accepts the following parameters:
 
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|api_key|credential_secret_key|None|False|urlscan API key. Not required for the search action|None|
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|api_key|credential_secret_key|None|False|urlscan API key. Not required for the search action|None|a46ec67a-0f2e7-c3879-26ac5d783ea4b8|
```
Comment (jschipp-r7) [inline urlscan/help.md:25]: Needs a raw example input section below the connection table

File: urlscan/help.md [hunk 3]
```diff
@@ -20,9 +20,19 @@ on potential malicous indicators.
...
+
+Example input:
+
+```
+{
+  "api_key": {
+    "secretKey": “9de5069-c5afe602b-2ea0a04b66beb2c0”
```
Comment (jschipp-r7) [inline urlscan/help.md:32]:
```suggestion
    "secretKey": “a46ec67a-0f2e7-c3879-26ac5d783ea4b8”
```

Comment (jschipp-r7) [reply urlscan/help.md:32]: We should keep these the same. We will auto-generate the raw JSON as well from the example key soon.

File: urlscan/plugin.spec.yaml [hunk 4]
```diff
@@ -141,35 +141,39 @@ connection:
     description: urlscan API key. Not required for the search action
     type: credential_secret_key
     required: false
+    example: a46ec67a-0f2e7-c3879-26ac5d783ea4b8
```
Comment (jschipp-r7) [inline urlscan/plugin.spec.yaml:144]: Doesn't the key format have 5 fields separated by 4 `-`'s?

Comment (mrinehart-r7) [reply urlscan/plugin.spec.yaml:144]: Use this one: 381cd93b-1946-9c73-1946-c916075eb9a3 (sample)

Comment (jschipp-r7) [reply urlscan/plugin.spec.yaml:144]: And regen

**Pull request merged**